### ISSUE TITLE
Improve screen reader compatibility

### ISSUE THREAD

**Issue by slannon97** [yuliskov/SmartYouTubeTV#207]
I am totally blind and use this app on my Fire TV Stick with the Voiceview screen reader. It makes navigating YouTube really easy and enjoyable. I am able to look at my subscriptions and easily subscribe/unsubscribe from channels. However, the app could work a little better with VoiceView. There are places where it says "button" or reads nothing at all. These controls should be properly labeled so that a blind user knows exactly what they will do. I also haven't found a way to get the screen reader to reliably read the video description and/or comments to me. Either this information isn't displayed on the screen or VoiceView cannot detect and read it. The following guide on Amazon's site will help. https://developer.amazon.com/docs/fire-tv/implement-accessibility-fire-os.html

Following the Android accessibility guidelines will also ensure the app is accessible across all Android TV platforms. I recommend testing the app with VoiceView so you get an idea of how it works. VoiceView can be toggled on or off on any Fire TV running the latest software update by pressing and holding the back and menu buttons on the remote or going to settings>accessibility>VoiceView. The guide I linked to above gives a very good overview of how VoiceView works.

If you have any questions, let me know and I will do my best to answer them. I hope you will make the app more screen reader friendly.

**Comment by slannon97** [comment]
Here's another guide which may help. I thought the previous link was this one. Either way, youwill have access to both. https://developer.amazon.com/docs/fire-tv/implement-voiceview-accessibility-features-fire-os.html

**Comment by yuliskov** [comment]
@slannon97 thanks for letting me know, friend. I'll return to this after I deal with other issues.
